tests/testsuite/test.rs

```diff
@@ -1346,6 +1346,7 @@ fn test_no_run() {
             "\
 [COMPILING] foo v0.0.1 ([CWD])
 [FINISHED] test [unoptimized + debuginfo] target(s) in [..]
+[EXECUTABLE] [..] (target/debug/deps/foo-[..][EXE])
```

Comment:
If you have a chance, I would appreciate if you can remove the `[..]` here and the other tests just so that the tests are verifying the actual output.